src/main/java/org/broadinstitute/hellbender/dev/tools/walkers/bqsr/BaseRecalibratorDataflow.java

```diff
@@ -118,7 +121,7 @@ protected void setupPipeline(Pipeline pipeline) {
             BaseRecalibratorDataflowUtils.ensureReferenceIsReadable(pipeline.getOptions(), referencePath);
```

Comment:
Earlier, on line 116, we say "saveTextualTables = (null!=BRAC.RAC.RECAL_TABLE_FILE)". We'll need to make RECAL_TABLE_FILE optional.